### ISSUE TITLE
Difference over the original?

### ISSUE THREAD

**Issue by aadamsx** [thepassle/create-lit-app-advanced#1]
It's not immideatly apparent how this repo is different or advanced over your original CLA: https://github.com/thepassle/create-lit-app

**Comment by thepassle** [comment]
Good point, I'm working on something of a doc site for create-lit-app that will highlight the differences between the 'standard' and the 'advanced' one as well. I've added loads of information to the readme, and its getting a little cluttered. I'm moving the demos/examples over the doc site, it's a work in progress that will take some more time, but im working on it.
